File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/dto/UserCaseDto.java
package org.javamaster.invocationlab.admin.dto;

import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * 用例详细信息
 *
 * @author yudong
 */
@EqualsAndHashCode(callSuper = true)
@Data
public class UserCaseDto extends AbstractCaseDto {
    String zkAddress;
    String serviceName;
    /**
     * 类路径,包含package和版本
     */
    String interfaceKey;
    /**
     * 接口的简单名称,eg:QueryProvider
     */
    String className;
    String methodName;
    String requestValue;
    String responseValue;
    String testScript;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/dto/UserCaseDto.kt
package org.javamaster.invocationlab.admin.model.dto

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * 用例详细信息
 *
 * @author yudong
 */

@AllOpen
class UserCaseDto : AbstractCaseDto() {
    var zkAddress: String? = null
    var serviceName: String? = null

    /**
     * 类路径,包含package和版本
     */
    var interfaceKey: String? = null

    /**
     * 接口的简单名称,eg:QueryProvider
     */
    var className: String? = null
    var methodName: String? = null
    var requestValue: String? = null
    var responseValue: String? = null
    var testScript: String? = null
}
